bin/dev.js

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ts-node
+// eslint-disable-next-line node/shebang
```

Comment:
why?

Reply:
eslint complains with `this file needs no shebang`, which isn't true